tools/testsys/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "testsys"
+version = "0.1.0"
+authors = ["Ethan Pullen <pullenep@amazon.com>", "Matt Briggs <brigmatt@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0"
+bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "cf5c765", version = "0.1"}
+bottlerocket-variant = { version = "0.1", path = "../../sources/bottlerocket-variant" }
+clap = { version = "3", features = ["derive", "env"] }
+futures = "0.3.8"
+k8s-openapi = { version = "0.14", features = ["v1_20", "api"], default-features = false }
+maplit = "1.0.2"
+model = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "cf5c765", version = "0.1"}
```

Comment:
Might be good to give `model` a more specific name, since we also have models in this repo.